### ISSUE TITLE
[DEP0091] DeprecationWarning: crypto.DEFAULT_ENCODING is deprecated.

### ISSUE THREAD

**Issue by azu** [textlint/textlint#620]
```
(node:68450) [DEP0091] DeprecationWarning: crypto.DEFAULT_ENCODING is deprecated.
(node:68450) [DEP0010] DeprecationWarning: crypto.createCredentials is deprecated. Use tls.createSecureContext instead.
(node:68450) [DEP0011] DeprecationWarning: crypto.Credentials is deprecated. Use tls.SecureContext instead.
```

Reproduce

- `node --trace-deprecation textlint README.md`

```
import * as crypto from "crypto"
```

This importing cause this warning.

For details, see https://github.com/nodejs/node/issues/23203

## Task

- [x] Fix this line #621 
- https://github.com/textlint/textlint/blob/318b954a830d39e24c52803b5307384512e5d5db/packages/textlint/src/config/config.ts#L3

**Comment by azu** [comment]
Help to debug option on Node.js

https://nodejs.org/api/cli.html#cli_throw_deprecation
https://nodejs.org/api/cli.html#cli_trace_deprecation
